webodf/lib/gui/SessionController.js

```diff
@@ -116,6 +116,13 @@ gui.SessionControllerOptions = function () {
             "Expected to be run in an environment which has a global window, like a browser.");
 
         /**
+         * @return {!gui.SessionControllerOptions}
+         */
+        this.getConfiguration = function() {
```

Comment:
Good catch, slipped me completely.
@peitschie This is from your single commit 40698e9cfd93564cccee80a5eb40da113e32c43a ("Expose SessionController configuration arguments for re-use"). Are you still using that in some way in your own code, and if so, can it need to be read-only?
For now I tend to propose to remove this commit simply.